### ISSUE TITLE
line() not drawing when end vertex is "too" out-of-bounds

### ISSUE THREAD

**Issue by easypeas** [processing/processing#5301]
## Description

```line()``` does not get drawn if the end vertex is too high. Notice how a lower out-of-bounds value is just fine.

## Expected Behavior

All lines drawn, or at least run-time error.

## Current Behavior

Seems like there is an arbitrary cutoff point for end vertices, after which the line does not get drawn

## Steps to Reproduce

```
size(100,100);
line(10, 10, 10, 2.14e9); // This line is drawn, no problem
line(20, 10, 20, 2.15e9); // This line is not drawn
line(30, 10, 30, 1/0.0); // This line to infinity is not drawn either, I can only see the start vertex
```

## Your Environment

 

* Processing version: 3.3.6
* Operating System and OS version: Windows 10
* Other information:

## Possible Causes / Solutions

I suspect some intermediate calculations go out of range. In particular, [calculation of the vertex norm](https://github.com/processing/processing/blob/master/core/src/processing/core/PGraphics.java#L1449) might require a ```double```, or special-casing of infinity. Thanks.

**Comment by gohai** [comment]
@easypeas Is there any real-world case where one would be hitting this behavior? Thanks.

**Comment by easypeas** [comment]
Yes, my very first Processing program. I was plotting (x-1)/(4-x^2), which goes to infinity at ±2.
